lib/linguist/languages.yml

```diff
@@ -2416,6 +2416,15 @@ Gleam:
   - ".gleam"
   tm_scope: source.gleam
   language_id: 1054258749
+Glimmer JS:
```

Comment:
Something to ponder... what will developers expect to see in the sidebar of their repos?

As this stands they'll see JavaScript because of the `group: JavaScript` line below. Individual files will however be classified as "Glimmer JS" and correctly highlighted.

If developers expect it to be called "Glimmer JS" then great, nothing to change other than to remove the `group` line. 

If they expect it to be called "Glimmer" as was the intent of the first PR then I recommend naming this "Glimmer".

I'm thinking about the future here. When "Glimmer TS" is added, will users expect to see it reflected as "Glimmer", "Glimmer JS" or "Glimmer TS" in the sidebar?

If "Glimmer", then it's best to name this entry "Glimmer" now and "Glimmer TS" can be grouped with a `group: Glimmer` line when it's added. This will result in both variants appearing as "Glimmer" in the sidebar but files will be individually identifed as the JS/TS variants.

If "Glimmer JS", then all is good and `group: Glimmer JS` can be included in the "Glimmer TS" entry.

If "Glimmer TS", then all is good and it can be added as a completely standalone language without grouping in future.

Reply:
🤔 I've definitely misconfigured the `group` key at present then.

My intuition is that Glimmer JS and Glimmer TS would be fine since it's helpful to see progress in a partially-typed repo. I'll reach out to the community for feedback, though.

Reply:
Consensus from the community:

| Language  | Group (for language summary) |
| ------------- | ------------- |
| Glimmer JS  | JavaScript  |
| Glimmer TS  | TypeScript  |

So I don't think any changes will be required.

When we open the Gimmer TS PR it will add its own language entry.